src/components/generateCars.ts

```diff
@@ -0,0 +1,39 @@
+import GarageAPI from './GarageAPI';
+
+async function generateCars() {
```

Comment:
I don't think it has correct name, generateCars sounds like an instant action but this code just setups an event handler